tsconfig.json

```diff
@@ -1,17 +1,28 @@
 {
   "compilerOptions": {
     "jsx": "preserve",
-    "module": "system",
+    "module": "ESNext",
```

Comment:
should it be capital case or lower case esnext

Reply:
does not matter